app/assets/stylesheets/adl-overrides/adl-overrides.scss

```diff
@@ -470,3 +470,9 @@ body.public-facing {
     object-fit: cover;
   }
 }
+
+// for catalog search result snippets
+.highlight {
+  background: yellow;
+  font-weight: 700;
+}
```

Comment:
This is the only meaningful change on this file, my auto linting plugin deleted all the trailing spaces